commands/compile/compile.go

```diff
@@ -61,10 +84,12 @@ func Compile(ctx context.Context, req *rpc.CompileReq, outStream, errStream io.W
 		fqbnIn = sketch.Metadata.CPU.Fqbn
 	}
 	if fqbnIn == "" {
+		stats.Incr("compile", stats.M(tags)...)
 		return nil, fmt.Errorf("no FQBN provided")
 	}
 	fqbn, err := cores.ParseFQBN(fqbnIn)
 	if err != nil {
+		stats.Incr("compile", stats.M(tags)...)
```

Comment:
I find a bit hard to read that `tags` here contains `success: false`, can we tag success status explicitly?